www/js/app.js

```diff
@@ -1060,7 +1092,9 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
      * @param {DirEntry} dirEntry The directory entry of the article to read
      */
     function readArticle(dirEntry) {
-	    // Only update for expectedArticleURLToBeDisplayed.
+        // Reset state.searches.lastPrefix to allow users to search the same string again if they want to
```

Comment:
Note to self: comment is incorrect, should read "Reset search prefix to allow...."